packages/tx/test/eip3860.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { Chain, Common, Hardfork } from '@ethereumjs/common'
+import { Address } from '@ethereumjs/util'
+import * as tape from 'tape'
+
+import { TransactionFactory } from '../src'
+
+const common = new Common({
+  chain: Chain.Mainnet,
+  hardfork: Hardfork.Merge,
+  eips: [3860, 4844, 4895],
+})
+
+const maxInitCodeSize = common.param('vm', 'maxInitCodeSize')
+const txTypes = [0, 1, 2, 5]
```

Comment:
No idea but that's what the EIP specifies.

Reply:
At the end you can't make an order for tx type numbers just out of EIPs, since there is no guarantee in which HF things are applied (if this is the goal to keep tx type numbers somewhat "chronologically").

So I would cautiously assume that there might just be other EIPs "in between" which claiming tx type numbers 3 and 4 (but totally just a guess).